winlogbeat/eventlog/wineventlog.go

```diff
@@ -218,7 +219,7 @@ func reportDrop(reason interface{}) {
 // newWinEventLog creates and returns a new EventLog for reading event logs
 // using the Windows Event Log.
 func newWinEventLog(options map[string]interface{}) (EventLog, error) {
-	var c winEventLogConfig
+	c := winEventLogConfig{BatchReadSize: defaultBatchReadSize}
```

Comment:
The more "ucfg" like approach here would be probably to introduce a `defaultConfig` and set detaulBatchReadSize there?